packages/server/src/cloud/fission/README.md

```diff
@@ -0,0 +1,162 @@
+# medplum-fission-test
+
+## Fission Router Configuration
+
+To interact with Fission functions from your application, you need to correctly configure the Fission Router's address. The `routerHost` and `routerPort` settings in your `FissionConfig` determine how your application connects to the Fission Router.
+
+These settings specify the network address (hostname or IP address) and port through which the Fission Router can be reached.
+
+- For applications running outside the Kubernetes cluster (e.g., local development machine, external client):
+  - `routerHost`: This should be the IP address of a Kubernetes Node, or `localhost` / `127.0.0.1` if you are using a local Kubernetes setup like Docker Desktop.
+  - `routerPort`: This must be the `NodePort` exposed by the Fission Router service. You can find this by running `kubectl get svc -n fission router`. In common Fission installations, this is often `31314`.
+- For applications running inside the Kubernetes cluster (e.g., another service in the same cluster):
+  - `routerHost`: This should be the internal ClusterIP service name of the Fission Router. Kubernetes DNS will resolve this name within the cluster. The standard format is `router.fission.svc.cluster.local`.
+  - `routerPort`: This will typically be the standard HTTP port `80`, as the internal ClusterIP service usually exposes the router on its default HTTP port.
+
+## Setup local dev
+
+https://fission.io/docs/installation/
+
+```bash
+export FISSION_NAMESPACE="fission"
+kubectl create namespace $FISSION_NAMESPACE
+kubectl create -k "github.com/fission/fission/crds/v1?ref=v1.21.0"
```

Comment:
Should `v1.21.0` be hardcoded here? Is there a more future-proof way to, e.g. get the newest v1 version? At the least, maybe `1.21.0` should be parameterized?